meta/Schema.php

```diff
@@ -145,6 +150,36 @@ public function __construct($table, $ts = 0, $islookup = false) {
     }
 
     /**
+     * Add the translatable keys to the configuration
+     *
+     * This checks if a configuration for the translation plugin exists and if so
+     * adds all configured languages to the config array.
+     *
+     * Adapted from @see \dokuwiki\plugin\struct\types\AbstractBaseType::initTransConfig
```

Comment:
An excellent question. I have no idea :-) Quick googling suggests traits are loaded through the same autoloading mechanism as classes, so it should work.